spec/views/layouts/application.html.erb_spec.rb

```diff
@@ -20,6 +20,7 @@
 
   context 'no content for nav present' do
     it 'displays only the logo' do
+      allow(view).to receive(:user_fully_authenticated?).and_return(false)
```

Comment:
Can you explain why this change is suddenly necessary? Would it make sense to include this in a common `before` block for the entire test suite rather than adding to it to every test case?